BrowserKit/Sources/Common/Theming/PrivateModeTheme.swift

```diff
@@ -13,17 +13,17 @@ public struct PrivateModeTheme: Theme {
 
 private struct PrivateModeColorPalette: ThemeColourPalette {
     // MARK: - Layers
-    var layer1: UIColor = FXColors.Ink50
-    var layer2: UIColor = FXColors.Ink50
+    var layer1: UIColor = FXColors.Violet90
```

Comment:
curious - what's the context on changing these design tokens if we're only adding a border here?

Reply:
When looking at the border I had implemented that it didn't look like the design and so I started checking the colors and noticed that some where not matching our Mobile Styles Figma, which is why I updated them. Should have probably be done in another PR to make it clearer.

Reply:
oh appreciate the context! hmm I wonder why they got updated 🤔 , but thanks for making this change!